raspberry-mint-dev/src/www/index.html

```diff
@@ -0,0 +1,669 @@
+                    Betty
+                </a>
+            </div>
+            <button id="toggle-nav-language" class="btn btn-primary toggle-nav" type="button"
+                    data-bs-toggle="modal"
+                    data-bs-target="#nav-language"
+                    aria-controls="nav-language" aria-expanded="false" aria-label="Toggle translations">
+                Translations
+            </button>
+            <button id="toggle-nav-search" class="btn btn-primary toggle-nav" type="button"
+                    data-bs-toggle="modal"
+                    data-bs-target="#nav-search"
+                    aria-controls="nav-search" aria-expanded="false" aria-label="Toggle search">
+                Search
+            </button>
+            <button id="toggle-nav-primary" class="btn btn-primary d-lg-none toggle-nav" type="button"
+                    data-bs-toggle="modal"
+                    data-bs-target="#nav-primary"
+                    aria-controls="nav-primary" aria-expanded="false" aria-label="Toggle primary nagivation">
+                Menu
+            </button>
+        </nav>
+
+        <nav class="navbar navbar-expand-lg p-0">
+            <div id="nav-primary" class="collapse navbar-collapse pb-lg-1">
+                <ul class="navbar-nav gap-4">
+                    <li class="nav-item active h6">
+                        <a class="nav-link px-lg-0" href="#">People</a>
+                    </li>
+                    <li class="nav-item h6">
+                        <a class="nav-link px-lg-0" href="#">Timeline</a>
+                    </li>
+                    <li class="nav-item h6">
+                        <a class="nav-link px-lg-0" href="#">Places</a>
+                    </li>
+                    <li class="nav-item h6">
+                        <a class="nav-link px-lg-0" href="#">Sources</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </nav>
+</div>
+<div id="nav-language" class="modal">
+    <div class="modal-dialog modal-sm modal-fullscreen-md-down">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">Language</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <nav class="modal-body">
+                <ul class="list-unstyled text-center">
+                    <li>
+                        <a href="#">English</a>
+                    </li>
+                    <li>
+                        <a href="#">français</a>
+                    </li>
+                    <li>
+                        <a href="#">Nederlands</a>
+                    </li>
+                    <li>
+                        <a href="#">українська</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+</div>
+<div id="nav-search" class="modal">
```

Comment:
Modals should have less horizontal padding when full-screen. Same as containers.